crisischeckin/CrisisCheckinMobile/CrisisCheckinMobile/LoginPage.cs

```diff
@@ -16,8 +17,9 @@ public class LoginPage : ContentPage
             if (isValid)
             {
                 //App.IsUserLoggedIn = true;
-                Navigation.InsertPageBefore(new ProfileView(), this);
-                await Navigation.PopAsync();
+                //Navigation.InsertPageBefore(new ProfileView(), this);
+                //await Navigation.PopAsync();
+                await Navigation.PushModalAsync(new Views.MainTabbedView(), true);
```

Comment:
I see. So would it make more sense to reverse what I did here?  In other words, make the Login screen the modal that is pushed onto the stack using PushModalAsync.  My understanding of Modal navigation is when one full screen view leads to another full screen view. No UI elements are shared between the original view and the new view. That way the user must interact with the login page before it is removed with PopModalAsync.